db/schema.rb

```diff
@@ -62,10 +62,10 @@
     t.integer  "gift_request_id"
     t.integer  "donor_id"
     t.string   "tracking_num"
-    t.boolean  "delinquent"
+    t.boolean  "delinquent",      default: false
     t.string   "state"
-    t.datetime "created_at",      null: false
-    t.datetime "updated_at",      null: false
+    t.datetime "created_at",                      null: false
```

Comment:
Yea, I might hold off on doing this because every time a `rake db:migrate` is called, in conjunction with everyone's personal Sublime/IDE whitespace settings sort of messes this up